composer.json

```diff
@@ -32,6 +34,15 @@
     },
     "sort-packages": true
   },
+  "extra": {
```

Comment:
This should be installed via packagist. https://packagist.org/packages/humbug/php-scoper

See the comment. https://github.com/google/web-stories-wp/pull/4540/files#r491043156

Either use `--ignore-platform-reqs` flag or change to the platform to 7.2. See [example](https://github.com/xwp/unsplash-wp/blob/develop/composer.json#L9-L13) from unsplash.

Reply:
We cannot change the platform to 7.2 as the plugin supports WordPress 5.6.

As I mentioned in the other thread, `--ignore-platform-reqs` only has downsides here.

PHP Scoper itself **recommends** using the PHAR installation method, see https://github.com/humbug/php-scoper#phar. Their example uses bamarni/composer-bin-plugin, which is similar to the one I use here.

Reply:
But if I install this locally and want to contribute and I am running say 7.1, then this will not work.

Reply:
For contributors we recommend a minimum required PHP version, just like we recommend the Node version.

It‘s reasonable to expect contributors to not run an outdated version of PHP IMO

Reply:
So what is the issue of including this as a composer dev dependency?

Reply:
Exactly the same reason as #4540